docs/quickstart/assessment/standalone_assessment.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2021 Learnosity, MIT License
```

Comment:
The LICENSE.md says Apache 2.0. Unless this is on purpose, we should bring this one in line.

Side note, for any new work anywhere, I strongly recommend using SPDX notation for licenses, as they are machine-extractable, so either 
```suggestion
# Copyright (c) 2021 Learnosity, MIT License
# SPDX-License-Identifier: MIT
```
or
```suggestion
# Copyright (c) 2021 Learnosity, Apache 2.0 License
# SPDX-License-Identifier: Apache-2.0
```